CHANGELOG.md

```diff
@@ -1,14 +1,33 @@
-## SuperTuxKart 0.9.4
-* Better random item distribution for various numbers of karts
-* Improved powerup handling in AI
-
-## SuperTuxKart 0.9.3
+# Changelog
+This file documents notable changes to SuperTuxKart across versions since its inception.
+
+It should be kept in mind that some versions have a less complete changelog than others, and that this changelog do not list the details of the many small bugfixes and improvements which together make a significant part of the progress between releases.
+
+For similar reasons, and because some features are vastly more complex than others, attributions of main changes should not be taken as a shorcut for overall contribution.
+
+## Unreleased
+* Better random item distribution for various numbers of karts by Alayan and hiker
```

Comment:
Network multiplayer is the big missing one, though I guess we can add it when it's closer to completion

Reply:
My reasoning is that Benau and hiker know this code much better than me and should be able to write a better changelog for it.